wagtail/snippets/static_src/wagtailsnippets/js/snippet-multiple-select.js

```diff
@@ -19,13 +19,13 @@ var updateDeleteButton = function(anySelected, newState) {
     });
     if (anySelected) {
         // enable button and add url
-        $deleteButton.removeClass('disabled');
+        $deleteButton.css('visibility', 'visible');
```

Comment:
I suggest using the `.visuallyhidden` class - https://github.com/wagtail/wagtail/blob/master/wagtail/admin/static_src/wagtailadmin/scss/core.scss#L65

Reply:
This way it will be consistent with other parts of the UI.

Reply:
Will do, thanks for pointing that out!